src/dotnet/MLSearch.Service/Indexing/ChatContent/ChatContentArranger3.cs

```diff
@@ -0,0 +1,202 @@
+using ActualChat.Chat;
+using ActualChat.Chat.ML;
+using ActualChat.MLSearch.Documents;
+
+namespace ActualChat.MLSearch.Indexing.ChatContent;
+
+internal sealed class ChatContentArranger3(
```

Comment:
is suffix `3` intentional? It's not saying much

Reply:
It's just another variant of ChatContentArranger. Eventually we will keep the only one.